diff --git a/node_modules/vue-toasted/dist/vue-toasted.js b/node_modules/vue-toasted/dist/vue-toasted.js
index 28d2061..d89c505 100644
--- a/node_modules/vue-toasted/dist/vue-toasted.js
+++ b/node_modules/vue-toasted/dist/vue-toasted.js
@@ -659,14 +659,14 @@ Object.defineProperty(__webpack_exports__, "__esModule", { value: true });
 
 
 var Toasted = {
-    install: function install(Vue, options) {
+    install: function install(app, options) {
         if (!options) {
             options = {};
         }
 
         var Toast = new __WEBPACK_IMPORTED_MODULE_0__js_toast__["a" /* Toasted */](options);
-        Vue.component('toasted', __WEBPACK_IMPORTED_MODULE_1__toast_vue___default.a);
-        Vue.toasted = Vue.prototype.$toasted = Toast;
+        app.component('toasted', __WEBPACK_IMPORTED_MODULE_1__toast_vue___default.a);
+        app.config.globalProperties.$toasted = Toast;
     }
 };
 
diff --git a/node_modules/vue-toasted/package.json b/node_modules/vue-toasted/package.json
index 4cd5760..6419bb3 100644
--- a/node_modules/vue-toasted/package.json
+++ b/node_modules/vue-toasted/package.json
@@ -8,7 +8,7 @@
     "type": "git",
     "url": "https://github.com/shakee93/vue-toasted.git"
   },
-  "main": "./dist/vue-toasted.min.js",
+  "main": "./dist/vue-toasted.js",
   "types": "./types/index.d.ts",
   "scripts": {
     "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
diff --git a/node_modules/vue-toasted/postcss.config.js b/node_modules/vue-toasted/postcss.config.js
deleted file mode 100644
index ccc98cd..0000000
--- a/node_modules/vue-toasted/postcss.config.js
+++ /dev/null
@@ -1,7 +0,0 @@
-var autoprefixer = require('autoprefixer')
-
-module.exports = {
-	plugins: [
-		autoprefixer({browsers: ['last 7 versions']})
-	]
-}
\ No newline at end of file
